.github/CODEOWNERS

```diff
@@ -9,7 +9,7 @@
 # Rules are matched bottom-to-top, so one team can own subdirectories
 # and another the rest of the directory.
 
-*                                       @DataDog/agent-e2e-testing
+*                                       @DataDog/agent-devx-loops 
```

Comment:
That also means members of the `agent-e2e-testing` team don't get notifications and can't review PRs, is the team still useful then ?

Reply:
Next step would be removing the github team. We can discuss spreading ownership to shared components and containers-integrations

Reply:
Actually there was only `agent-e2e-testing`, so why would people ask for review in devx-loops ? 😕
Is it more in datadog-agent where there is a lot of double ownership ?

Reply:
The official ownership of the framework moved to Agent DevX Loops. We currently have one support channel specific to e2e tests, and one channel for reviews at Agent DevX Loops. 

Let's discuss on Slack!